src/dotnet-scaffolding/dotnet-scaffold-aspire/Program.cs

```diff
@@ -24,6 +24,7 @@
        .WithOption(appHostProjectOption)
        .WithOption(projectOption)
        .WithOption(prereleaseOption)
+       .WithStep<FirstTimeTelemetrySetupStep>()
```

Comment:
I think I see a couple of problems with this approach, but let me know if I'm missing something or overthinking it.

One, having to add this to each scaffolder makes it kind of easy to miss one. If a new scaffolder is added, but the person writing it doesn't remember to add this step, then someone could run the tool without the first notice experience.

Two, I wonder about other scenarios where we need to run the first run experience. what about `dotnet-scaffold-aspire --version`? Or `dotnet-scaffold-aspire --help`? If we collect telemetry for them, we'd need to display the banner. Right now we don't, so we could avoid it I guess. And I note that `dotnet` itself doesn't display it for things like `dotnet --info` and `dotnet --version`. But I'm guessing that's because they don't collect telemetry for them. 

I wonder if it would be simple enough to call something down before the runner.RunAsync (after the Build, so that the services are created) so that we guarantee it always executes no matter what the call is?

Reply:
yep I ran into the help scenario as well and thought it was fine to not show banner or collect telemetry then. I do prefer just doing it before RunAsync, and the adding the steps is cumbersome/easy to miss. Will switch to that, tyty

Reply:
addressed